release-post.md

```diff
@@ -0,0 +1,68 @@
+# Universal Sierra Compiler
+
+The Software Mansion Team has created a new tool, Universal Sierra Compiler (further abbreviated as USC).
+
+USC is the tool for Sierra compilation - it compiles any ever-existing Sierra version to CASM.
+
+The idea behind USC is simple, yet it might make things easier for some tools.
+It bundles all Sierra compilers previously released, and depending on the version contained inside the Sierra JSON, it chooses the right compiler.
+Therefore it allows any tool to be independent of Sierra version.
+
+Universal Sierra Compiler might become especially useful for nodes, which have to interact with contracts deployed in different versions of Sierra, but also any other tool that depends on the versioning of Sierra.
+
+
+## Usage
+
+| Command          | Purpose                               | Input                        | Input as JSON of Cairo struct                              | Output as JSON of Cairo struct |
+|------------------|---------------------------------------|------------------------------|------------------------------------------------------------|-|
+| compile-contract | compile Sierra of a Starknet contract | output of `cairo-compile`    | cairo_lang_starknet_classes::contract_class::ContractClass | cairo_lang_starknet::casm_contract_class::CasmContractClass |
```

Comment:
`cairo_lang_starknet_classes::contract_class::ContractClass` 
This is prone to changing so i would think about leaving those specifics out
Listing a tool that outputs the json in desired format is much more useful

Reply:
Thought it'd be easier to integrate for existing tools